File: Solutions/Python/length_of_last_word.py
"""
    [EASY]
    58. Length of Last Word

    Concepts:
    - string

    Stats:
        Runtime | 0 ms      [Beats 100%]
        Memory  | 16.63 MB  [Beats 100%]
"""

class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        return len(s.split()[-1])

File: Solutions/Rust/length_of_last_word.rs
/*
    [EASY]
    58. Length of Last Word

    Concepts:
    - string

    Stats:
        Runtime | 0 ms      [Beats 100%]
        Memory  | 2.22 MB   [Beats 16.46%]
*/
impl Solution {
    pub fn length_of_last_word(s: String) -> i32 {
        return s.trim().split(" ").last().unwrap().len() as i32;
    }
}